test/test_verify_command.rb

```diff
@@ -27,4 +27,32 @@ def test_verify
     assert_equal "Signed by non-maintainer: someone@example.org", output.shift
     assert_equal [], output
   end
+
+  def test_maintainer_signature_and_non_maintainer_signature
+    @cmd.options[:args] = [@gem_path]
+    uuids = ["maintainer_entry_uuid", "dummy_entry_uuid"]
+    stub_rekor_search_index_by_digest(returning: uuids)
+    stub_rekor_get_rekords_by_uuid(
+      uuids: uuids,
+      returning: {
+        uuids.first => {
+          cert_options: {
+            email: "rubygems.org@n13.org",
```

Comment:
It's the email that is baked into the fixture gem we're using: hello-world.gem.  We have not built our own test gem(s) yet.

Reply:
> It's the email that is baked into the fixture gem we're using: hello-world.gem. We have not built our own test gem(s) yet.

It might be worthwhile adding that as a comment so future us will know why this email gets used.